### PR TITLE
enh: If there is only one ticket type, please pre-select '1' instead of '0'

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5449

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/public/ticket-list.hbs [hunk 1]
```diff
@@ -88,12 +88,19 @@
...
+                      <div class="item" data-value="{{map-value mapper 1}}">1</div>
+                      {{#each (range 2 (add ticket.maxOrder 1)) as |count|}}
+                        <div class="item" data-value="{{map-value mapper count}}">{{count}}</div>
+                      {{/each}}
+                    {{else}}
+                      <div class="item" data-value="{{map-value mapper 0}}">0</div>
+                      {{#each (range ticket.minOrder (add ticket.maxOrder 1)) as |count|}}
+                        <div class="item" data-value="{{map-value mapper count}}">{{count}}</div>
+                      {{/each}}
+                    {{/if}}
```
Comment (iamareebjamal) [inline app/templates/components/public/ticket-list.hbs:103]: No need of this

File: app/templates/components/public/ticket-list.hbs [hunk 2]
```diff
@@ -88,12 +88,19 @@
                     @id={{concat ticket.id "_quantity"}}
                     @value={{ticket.orderQuantity}} />
                   <i class="dropdown icon"></i>
-                  <div class="default text">0</div>
+                  <div class="default text">{{if (eq this.tickets.length 1) 1 0}}</div>
```
Comment (iamareebjamal) [inline app/templates/components/public/ticket-list.hbs:91]: Have you verified that it actually selects 1 in code as well and not just in UI

Comment (maze-runnar) [reply app/templates/components/public/ticket-list.hbs:91]: No it doesn't select 1, for that part I have added the below logic.

Comment (maze-runnar) [reply app/templates/components/public/ticket-list.hbs:91]: just starting mapping from 1
```
{{#each (rannge 2 (add ticket.maxOrder 1)) as |count|}}
          <div class="item" data-value="{{map-value mapper count}}">{{count}}</div>
     {{/each}}
```

**Review by daretobedifferent18 — COMMENTED** [review]
Also you need to take care of the fact that 
```handlebars 
{{action "updateOrder" ticket}}
```
this only get triggered onChange

**Comment by daretobedifferent18** [comment]
@maze-runnar This issue is being solved Pls close this PR

**Pull request closed without merging**